python/pyspark/pandas/frame.py

```diff
@@ -6901,6 +6904,16 @@ def sort_values(
         3     D     7     2
         2  None     8     4
 
+        Ignore index for the resulting axis
+
+        >>> df.sort_values(by=['col1'], ignore_index=True)
+           col1  col2  col3
+        0     A     2     0
+        1     B     9     9
+        2     C     4     3
+        3     D     7     2
+        4  None     8     4
```

Comment:
Maybe can we refine the `df` by manually creating the index to show how `ignore_index` works more clearly??

e.g.
```python
df = ps.DataFrame({
    'col1': ['A', 'B', None, 'D', 'C'],
    'col2': [2, 9, 8, 7, 4],
    'col3': [0, 9, 4, 2, 3],
  },
  index=['idx1', 'idx2', 'idx3', 'idx4', 'idx5'],
  columns=['col1', 'col2', 'col3'])
```

Seems like the current example shows the same result regardless of `ignore_index` value.

Reply:
Thanks @itholic . The resulting index order is different considering different `ignore_index` input.
```py
        Sort by col1

        >>> df.sort_values(by=['col1'])
           col1  col2  col3
        0     A     2     0
        1     B     9     9
        4     C     4     3
        3     D     7     2
        2  None     8     4

        Ignore index for the resulting axis

        >>> df.sort_values(by=['col1'], ignore_index=True)
           col1  col2  col3
        0     A     2     0
        1     B     9     9
        2     C     4     3
        3     D     7     2
        4  None     8     4
```
I may modify it if you still think it confusing though.

Reply:
Ohh.. I see. Then I think we can keep this as is.

Reply:
Modified @itholic  :)

Reply:
Great!